src/libraries/Common/src/Interop/Browser/Interop.Runtime.cs

```diff
@@ -15,6 +15,8 @@ internal static unsafe partial class Runtime
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern void InvokeJSFunction(IntPtr bound_function_js_handle, void* data);
         [MethodImpl(MethodImplOptions.InternalCall)]
+        public static extern void InvokeImportedFunction(IntPtr fn_handle, void* data);
```

Comment:
Why the new very similar icall?

Reply:
The other one is for marshaled delegates, which have normal JSHandle. Imported functions are not JSObject after this change and don't have JSHandle.

Reply:
And we need another code path, to resolve it from different collection.

Reply:
I see, so in this case the former is 'JS Function' and the latter is actually an import. The overload of the term 'function' there is a little confusing but it's probably fine